seed/index.js

```diff
@@ -0,0 +1,48 @@
+const mongoose = require('mongoose');
+const _ = require('underscore');
+
+// connect to mongodb
+mongoose.connect('mongodb://localhost/airbnb', {useNewUrlParser: true, useUnifiedTopology: true});
+
+const db = mongoose.connection;
+
+db.on('error', console.error.bind(console, 'connection error: '));
+db.once('open', function() {
+  console.log('We are connected!');
+});
+
+// define more places to stay schema
+const morePlacesSchema = new mongoose.Schema({
+  morePlaces: [Number],
+  propertyId: Number
+});
+
+// compile schema into model
+const MorePlaces = mongoose.model('MorePlaces', morePlacesSchema);
+
+const seedMorePlaces = async function() {
+  await MorePlaces.deleteMany({});
+
+  const range = _.range(1, 101);
+  _.each(range, async (element, index, list) => {
```

Comment:
will the index and list variables be used?

Reply:
@GalexyN no they will not be used. i was following underscore documentation syntax. i think i can remove those.  thanks for flagging